__tests__/model/model.query.test.js

```diff
@@ -21,7 +22,8 @@ describe('Product queries', () => {
       variables: { category: newCategory },
     });
     categoryId = createCategory.data.addCategory._id;
-    categoryName = createCategory.data.addCategory.name;
+   
+    // categoryName = createCategory.data.addCategory.name;
```

Comment:
Please, remove useless code

Reply:
Removed.
I know that this code was useless, but this component is not mine, so I commented it instead of completely remove it